src/components/scenes/CreateWalletSelectCryptoScene.js

```diff
@@ -127,21 +134,29 @@ export class CreateWalletSelectCrypto extends Component<Props, State> {
   }
 
   static getDerivedStateFromProps (nextProps: Props, prevState: State) {
+    const { displayErrorAlert } = nextProps
+    let { errorShown } = prevState
     // Sort the wallet types
     const sortedWalletTypes: Array<GuiWalletType> = []
     const walletTypesCopy: Array<GuiWalletType> = [].concat(nextProps.supportedWalletTypes)
-
+    let unloadedWalletCount = 0
     for (const wt of WALLET_TYPE_ORDER) {
       const idx = walletTypesCopy.findIndex(gwt => gwt.value === wt)
       if (idx >= 0) {
         sortedWalletTypes.push(walletTypesCopy[idx])
         walletTypesCopy.splice(idx, 1)
       } else {
-        throw new Error('Error missing wallet type. Ensure all plugins are loaded')
+        unloadedWalletCount++
+      }
+    }
+    if (unloadedWalletCount) {
+      if (!errorShown) {
+        displayErrorAlert('Error missing wallet type. Unloaded plugins.')
```

Comment:
Error message should be localized in en_US.js

Also recommend a more user friendly message.
"Some wallet types unavailable. Please wait for account to fully load to access all wallet types."

Reply:
I'll try it and see how much of that message can fit in the small dropdown alert.